.changeset/cute-taxis-double.md

```diff
@@ -0,0 +1,18 @@
+---
+'@backstage/backend-defaults': minor
+'@backstage/integration': minor
+---
+
+**BREAKING** Bitbucket cloud now supports API token as App password will be deprecated soon.
+
+```yaml
+integrations:
+  bitbucketCloud:
+    - host: bitbucket.org
+      # Option 1 (recommended): API Token
+      token: ${BITBUCKET_API_TOKEN}
```

Comment:
```suggestion
    # Option 1 (recommended): API Token
    - token: ${BITBUCKET_API_TOKEN}
```
`host` is obsolete/static. It cannot be another value. There is no reason for the user to be able to set a value.